### ISSUE TITLE
`prpy` is unusable without `ros_control_client_py`

### ISSUE THREAD

**Issue by psigen** [personalrobotics/prpy#307]
The latest `master` of prpy cannot be imported at all if `ros_control_client_py` is not installed.
```
import prpy
-> import controllers
-> from rewd_controllers import OrController, RewdOrController, RewdOrTrajectoryController
-> from ros_control_client_py import FollowJointTrajectoryClient, TrajectoryExecutionFailed
```

Is it really necessary to put this in `prpy/__init__.py`?
I feel like it is reasonable for people to selectively import controllers when they need them.

**Comment by mkoval** [comment]
@ClintLiddick agree that we should definitely make this an optional dependency. Someone in lab got burned by the same issue, so this is high on @ClintLiddick's priority list.

**Comment by mkoval** [comment]
Fixed by #308.
